elements/pf-popover/pf-popover.ts

```diff
@@ -174,114 +178,137 @@ export class PopoverShownEvent extends ComposedEvent {
 export class PfPopover extends LitElement {
   static readonly styles = [styles];
 
+  // todo: alertSeverityScreenReaderText
+  // todo: hideOnOutsideClick
+  // todo: withFocusTrap
+
   @property({ type: String, reflect: true }) position: Placement = 'top';
   @property({ type: String, reflect: true }) heading?: string;
   @property({ type: String, reflect: true }) body = '';
   @property({ type: String, reflect: true }) footer?: string;
   @property({ type: String, reflect: true }) icon?: string;
   @property({ type: String, reflect: true }) label?: string;
+  @property({ type: Number, reflect: true }) distance?: number = 25;
+  // todo: handle PF4s 'flip' Placement option in flip-behavior
+  // https://www.patternfly.org/v4/components/popover#popover
+  @property({ type: String, reflect: true, attribute: 'flip-behavior' }) flipBehavior?: Placement[] = ['top', 'bottom', 'left', 'right', 'top-start', 'top-end', 'bottom-start', 'bottom-end', 'left-start', 'left-end', 'right-start', 'right-end'];
```

Comment:
We should avoid accepting and certainly reflecting array props. If this *must* be a list, use a converter like the NumberListConverter